addons/source-python/docs/source-python/source/developing/module_tutorials/menus.rst

```diff
@@ -0,0 +1,117 @@
+Menus
```

Comment:
This should be all lowercase, so that the presentation is consistent with the other tutorials on the index:

![Index](https://i.imgur.com/NfqtO5p.png)